_data/apps.yml

```diff
@@ -265,3 +265,13 @@
     version: 5
     description: >
       SwingTradeBot's core application is powered by Rails and a PostgreSQL database.
+
+- name: Podia
+  url: https://www.podia.com
+  logo_url: https://www.dropbox.com/s/srkqnjlm2xgzopu/podia-wm-purple-medium.png
```

Comment:
I think Dropbox is rendering this image within an HTML page, so it's not loading correctly when required as an image. OK if we replace it with the logo from Twitter?

`https://pbs.twimg.com/profile_images/938423381891850240/OSO-VUJY_400x400.jpg`
![twitter](https://pbs.twimg.com/profile_images/938423381891850240/OSO-VUJY_400x400.jpg)

Reply:
@alextaylor000 yeah that would be perfect thanks for the heads up. Want me to update the URL?